inputs/orchestrator_sources_inner:6.json

```diff
@@ -0,0 +1,15 @@
+{
+  "prebuild_plugins": [
+    {
+      "name": "reactor_config"
+    }
```

Comment:
Is including fetch_sources a separate change?

Reply:
Yes I'm tracking missing parts here: https://github.com/containerbuildsystem/osbs-client/pull/885

Firstly I want to debug/update all the inner/core parts of osbs-client and atomic-reactor. When this is sort of stable then I'll start adding plugins (note that dev testing with enabled fetch_sources become harder due needs of SRPMS and builds in koji)

Reply:
OK, thanks.